### ISSUE TITLE
Links must be distinguishable without relying on color

### ISSUE THREAD

**Issue by vh8154** [pulibrary/DSS#538]
### Steps to recreate
Run Axe DevTools on /catalog.

### URL
localhost:3000/catalog

### Issues
color-contrast

### Severity
High

### WCAG Success Criterion
[1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color)

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Not everyone perceives colors the same way, with some users having difficulty due to partial sight, color blindness, or monochrome displays. To improve accessibility, information should be conveyed through text cues or other visual indicators, not just color.

### Recommended fix
To solve this problem, you need to fix at least (1) of the following:
1) The link has insufficient color contrast of 2.53:1 with the surrounding text. (Minimum contrast is 3:1, link text: #2762ae, surrounding text: #212529)
2) The link has no styling (such as underline) to distinguish it from the surrounding text

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
Related to #526
